src/main/java/org/opentripplanner/middleware/models/MonitoredTrip.java

```diff
@@ -173,6 +174,10 @@ public class MonitoredTrip extends Model {
      */
     public boolean notifyAtLeadingInterval = true;
 
+    public RelatedUser primary;
+    public RelatedUser companion;
+    public List<RelatedUser> observers = new ArrayList<>();
```

Comment:
Note that these don't/no longer include a user id, so we would have to lookup the user id.

Reply:
I'm assuming updates are still to come? In a diff PR? All three parameters are currently unused.